lib/kinesis_client/stream/app_state/dynamo.ex

```diff
@@ -149,10 +149,10 @@ defmodule KinesisClient.Stream.AppState.Dynamo do
       return_values: "UPDATED_NEW"
     ]
 
-    case Dynamo.update_item(app_name, %{"shard_id" => shard_id}, update_opt) |> ExAws.request() do
+    case Dynamo.update_item(app_name, %{"shard_id" => shard_id}, update_opt)
+         |> ExAws.request() do
       {:ok, %{"Attributes" => %{"completed" => %{"BOOL" => true}}}} -> :ok
       {:error, {"ConditionalCheckFailedException", _}} -> {:error, :lease_owner_match}
-      reply -> reply
```

Comment:
Why did we remove this fallback match clause?

Reply:
Because if it fails, I'd like to know why and fix it. We can restore this if we're worried it's going to make it to obrittle.